src/helpers/uploadFiles.ts

```diff
@@ -26,7 +26,7 @@ export async function uploadFiles(
           fileName: `${timeStamp}-${f.name.replace(SPACES, "_")}`,
         }),
         headers: { authorization },
-      })
+      }).catch((e: any) => console.log(e))
```

Comment:
>  The upload API failure leads to abrupt crash of the update function

@abbasali1994, I tried to reproduce this error by just throwing error inside `uploadFiles.ts`

```ts
export async function uploadFiles(
  files: File[],
  bucket: Bucket
): Promise<string | null> {
  throw new Error("throw error thrown from upload files");
```

and error is catched by caller as expected ( no crash )
<img width="949" alt="image" src="https://user-images.githubusercontent.com/89639563/231076346-c4d5db95-e8ea-461c-9efa-633439534b4b.png">

also edit profile content is not cleared as stated in issue #2031

Reply:
Like I mentioned earlier the issue seems intermittent , even I was able to reproduce it only once after multiple tries, this was just a safeguard I thought would be best to avoid that, but based on your input the catch clause does seem redundant, I will put the PR in draft for now